lib/service_check.pm

```diff
@@ -229,6 +229,9 @@ Check service before migration, zypper install service package, enable, start an
 =cut
 sub install_services {
     my ($service) = @_;
+    # trun off lmod shell debug information
+    assert_script_run('echo export LMOD_SH_DBG_ON=1 >> /etc/bash.bashrc.local');
```

Comment:
Need we workaround this for all ARCH, or just ARM and x86_64?

Reply:
currently it's caused by hpc modules, we can leave it here and I can run on power and 390 to verify it will not broken
them.

Reply:
Do you have verify log on s390 and power?

Reply:
verify on s390x and power
http://openqa.nue.suse.com/t5677549 s390x https://openqa.nue.suse.com/t5685677
http://openqa.nue.suse.com/t5677548 ppc64 https://openqa.nue.suse.com/t5685676